### DISCUSSION TITLE
Sorting attachments within a task by date

### DISCUSSION THREAD

**Discussion by arocholl** [kanboard/kanboard#5176] [General]
I am considering myself doing the code changes required to sort attachment listed in a task, currently this is not available and hard to review when there is large list of files attached.

So my question here is whether someone suggest this being easier to implement as a plugin or as a core code change. I am not yet contributed to kanboard code so I am not familiar with that.

Thanks for your input.

**Comment by NDEEKB** [comment]
My organization would also be interested in being able to sort the attachments by date rather than name.  Attachments are very difficult to find once uploaded into a large list.  I am not a coder and unable to do this myself but have staff who might be capable of it.  This issue seems to need an **ENHANCEMENT** label.  Thanks and have a great day! DB
